test/utils/BatchCall.test.js

```diff
@@ -0,0 +1,33 @@
+const { BN, expectRevert } = require('@openzeppelin/test-helpers');
+const MulticallTokenMock = artifacts.require('MulticallTokenMock');
+
+contract('MulticallToken', function (accounts) {
```

Comment:
We're missing two important tests:

- A revert in one of the subcalls will revert all of the state changes of previous subcalls.
- The return value is an array of the return values of subcalls, in the order they were specified.